### PR TITLE
Update query filtering examples 

### PR DESCRIPTION
**Pull request by sunker**
Now that [query filtering](https://github.com/grafana/grafana/pull/84656) is being changed in Grafana, it's time to update our examples. There's no longer a need to filter out hidden queries in the data source as this is being taken care of by Grafana. Also, `filterQuery` is now being called from the query runner, so frontend data sources no longer need to do this in the query method. 

While at it, I'm also replacing usage of `defaults(target, defaultQuery)` with the implementation of `getDefaultQuery`.

### CODE REVIEW

File: examples/app-basic/yarn.lock
Comment (andresmgot) [inline examples/app-basic/yarn.lock:1]: why are there updates in the `yarn.lock`s?

File: examples/datasource-streaming-websocket/streaming-websocket-plugin/src/DataSource.ts [hunk 1]
```diff
@@ -65,9 +67,6 @@ export class DataSource extends DataSourceApi<MyQuery, MyDataSourceOptions> {
   }
 
   filterQuery(query: MyQuery): boolean {
-    if (query.hide) {
-      return false;
-    }
-    return true;
+    return !!query.queryText;
```
Comment (andresmgot) [inline examples/datasource-streaming-websocket/streaming-websocket-plugin/src/DataSource.ts:70]: I think queryText is not used in this plugin

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (andresmgot) [inline yarn.lock:1]: this file is probably not needed

**Review by andresmgot — APPROVED** [review]
LGTM

File: examples/datasource-streaming-websocket/streaming-websocket-plugin/src/DataSource.ts (hunk 1 above)
Comment (sunker) [reply examples/datasource-streaming-websocket/streaming-websocket-plugin/src/DataSource.ts:70]: Maybe it shouldn't be, but it is [defined](https://github.com/grafana/grafana-plugin-examples/blob/375cc7ec9eae4f7d712172b13a361ef6818d0377/examples/datasource-streaming-websocket/streaming-websocket-plugin/src/types.ts#L4) in types.

**Pull request merged**